code_review/16-09-20_s2s_util.py

```diff
@@ -0,0 +1,129 @@
+"""
+Utility functions for seq2science
+"""
+import os
+from typing import List
+
+import pandas as pd
+import pysradb
+from snakemake.io import expand
+
+
+def _sample_to_idxs(df: pd.DataFrame, sample: str) -> List[int]:
+    """
+    Get the index/indices that belong to a run in a pysradb dataframe
+    """
+    if sample.startswith(("SRR", "DRR", "ERR")):
+        idxs = df.index[df.run_accession == sample].tolist()
+        assert len(idxs) == 1, f"sample {sample} with idxs: {idxs}"
+    elif sample.startswith("SRX"):
+        idxs = df.index[df.experiment_accession == sample].tolist()
+        assert len(idxs) >= 1, len(idxs)
+    else:
+        assert False
```

Comment:
Do you want a message for the `assert`s for when they fail?

Reply:
I wasn't sure if I should add this assert even, because, in theory, this is impossible to reach (since this check is also already before).